ood-portal-generator/spec/fixtures/ood_portal.yaml.all

```diff
@@ -217,6 +217,24 @@ node_uri: '/configured-node'
 # Default: null (disable this feature)
 rnode_uri: '/configured-rnode'
 
+# Naive SSL: This path will force the upstream connection to use HTTPS or
+# WSS, used for backend applications that are running their own SSL termination
+# with self-signed certificates. Sub-uri used to reverse proxy to backend web
+# server running on node that knows the full URI path
+# Example:
+#     nodes_uri: '/nodes'
+# Default: null (disable this feature)
+nodes_uri: '/configured-nodes'
```

Comment:
I think `nodes_uri` is too close to `node_uri`. I'd prefer maybe `secure_node_uri` or `tls_node_uri` or similar so folks don't get tripped up on something so similar.

Applicable to `rnodes_uri` too.